resources/metadata/batsExecuteTests.yaml

```diff
@@ -0,0 +1,62 @@
+metadata:
+  name: batsExecuteTests
+  description: This step executes tests using the [Bash Automated Testing System - bats-core](https://github.com/bats-core/bats-core).
+  longDescription: |
+    Bats is a TAP-compliant testing framework for Bash. It provides a simple way to verify that the UNIX programs you write behave as expected. A Bats test file is a Bash script with special syntax for defining test cases. Under the hood, each test case is just a function with a description.
+spec:
+  inputs:
+    resources:
+      - name: tests
+        type: stash
+    params:
```

Comment:
groovy-based step was able to read a parameter `envVars` which was then passed to the step execution.

We should keep this behavior although it is not possible 1:1.

Suggestion:
- introduce multi value parameter `envVars` as `[]string`
- inject the envVars into the command execution of bats as environment variables
- format of parameter value could be `<KEY>=<VALUE>`